### ISSUE TITLE
I could not find a separate test split file or a train/validation/test split definition.

### ISSUE THREAD

**Issue by PatricKk0226** [giorgioskij/stage#2]
Hi, thank you for releasing the STAGE code and checkpoints.

I have a question about the data split used for the evaluation in the paper, especially Table 2 in Section 5.2.

In the paper, the accompaniment generation results are reported on the MoisesDB test set. However, in the released code, I only found a train/validation split in `stemmed_dataset.py`, where the last `N_VALID_SAMPLES = 24` songs are used as the validation set:

```python
if self.train:
    self.song_names = self.song_names[:-N_VALID_SAMPLES]
else:
    self.song_names = self.song_names[-N_VALID_SAMPLES:]

Could you please clarify how the MoisesDB test set used in Table 2 was defined?

Specifically:

1\Was the reported test set the same as the validation split defined in the released code, i.e., the last 24 sorted songs?
2\Or did you use a separate held-out test split that is not included in the repository?
3\If a separate test split was used, would it be possible to share the list of song IDs or the split file, so that we can reproduce the evaluation protocol more faithfully?

Thanks for your help!
